config/shared/activeUsers.js

```diff
@@ -0,0 +1,7 @@
+const activityTimeout = 60000;
```

Comment:
You are missing ```'use strict'``` directive

Also all file names in this folder are in ```kebab-case```, so please follow that convention